src/stores/widgets/StopGapWidgetDriver.ts

```diff
@@ -366,4 +368,50 @@ export class StopGapWidgetDriver extends WidgetDriver {
             client.off(ClientEvent.TurnServersError, onTurnServersError);
         }
     }
+
+    public async readEventRelations(
+        eventId: string,
+        roomId?: string,
+        relationType?: string,
+        eventType?: string,
+        from?: string,
+        to?: string,
+        limit?: number,
+        direction?: 'f' | 'b',
+    ): Promise<IReadEventRelationsResult> {
+        const client = MatrixClientPeg.get();
+        const dir =
+            direction && Object.values(Direction as Record<string, string>).includes(direction)
+                ? direction as Direction
+                : undefined;
```

Comment:
`direction` should already be compatible with the `Direction` enum, which means this shouldn't be needed as a check.

Reply:
I added it as a safeguard since the type guards of typescript are relatively weak hints considering this value was provided over the post message channel. On the other hand, we don't really do a similar input validation for the other fields so we could as well skip it here.